### PR TITLE
switch to shotover org cql-ws repo

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

**Review by rukai — APPROVED** [review]
I was hoping to get the cql-ws repo in a better state before merging this, but seeing we are blocked on ben who seems to be out today, lets go ahead and land it as its still an improvement over the previous state of things.

**Review by cjrolo — APPROVED** [review]

**Pull request merged**
